src/_pytest/nodes.py

```diff
@@ -173,10 +173,12 @@ def listchain(self):
         chain.reverse()
         return chain
 
-    def add_marker(self, marker):
+    def add_marker(self, marker, append=True):
```

Comment:
Nice 👍 

This will need to go to `features` though.

Reply:
should we split the addition of append=True then?

Reply:
I think so, unfortunately.

Reply:
While discussing API, should we introduce a `prepend_marker` instead of adding a boolean flag? I don't have a strong opinion, just thought I would mention.

Reply:
its in line with stuff like python warnings, so i'd like to keep

Reply:
Sure, no problem.